packages/core/src/ui/UIUtils.ts

```
@@ -0,0 +1,152 @@
+      (viewE[12] = -transform.position.x), (viewE[13] = -transform.position.y);
+      Matrix.multiply(virtualCamera.projectionMatrix, virtualCamera.viewMatrix, virtualCamera.viewProjectionMatrix);
+      renderContext.applyVirtualCamera(virtualCamera, false);
+      renderContext.rendererUpdateFlag |= ContextRendererUpdateFlag.ProjectionMatrix;
+      uiRenderQueue.clear();
+      uiCanvas._prepareRender(renderContext);
+      uiRenderQueue.pushRenderElement(uiCanvas._renderElement);
+      batcherManager.batch(uiRenderQueue);
+      uiRenderQueue.render(renderContext, "Forward");
+    }
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)